espnet2/gan_codec/shared/quantizer/residual_vq.py

```diff
@@ -83,9 +89,15 @@ def forward(
         """
         bw_per_q = self.get_bandwidth_per_quantizer(sample_rate)
         n_q = self.get_num_quantizers_for_bandwidth(sample_rate, bandwidth)
-        quantized, codes, commit_loss = self.vq(x, n_q=n_q)
+        quantized, codes, commit_loss, quantization_loss = self.vq(x, n_q=n_q)
         bw = torch.tensor(n_q * bw_per_q).to(x)
-        return quantized, codes, bw, torch.mean(commit_loss)
+        return (
+            quantized,
+            codes,
+            bw,
+            torch.mean(commit_loss),
+            torch.mean(quantization_loss),
+        )
```

Comment:
Could you please verify that the change won't introduce potential issues to the soundstream/encodec modules?